app/blog/page.tsx

```
@@ -0,0 +1,464 @@
+          }
+          if (cancelled) return
+          setViewPost(post)
+        } else {
+          if (cancelled) return
+          setViewPost(null)
+        }
+      } catch {
+        if (cancelled) return
+        setError('Failed to load blog')
+      } finally {
+        if (cancelled) return
```

Biome (2.4.4)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)